components/layout_2020/display_list/mod.rs

```diff
@@ -188,6 +190,38 @@ impl DisplayList {
             display_list: self,
         };
         fragment_tree.build_display_list(&mut builder, root_stacking_context);
+
+        if let Some(highlighted_dom_node) = highlighted_dom_node {
+            // Draw highlights around the node that is currently hovered in the devtools
+            // FIXME: Highlight margin and padding too
```

Comment:
This should be doable here, I think. I believe that we

Reply:
Margin, padding and border areas are now highlighted as well:

![image](https://github.com/user-attachments/assets/d31fda7b-c114-4c73-b2c4-29944de5ed66)

Reply:
(using roughly the same colors as firefox)